app/src/main/java/uc/edu/klopfsea/groupscheduler/model/EventList.kt

```diff
@@ -4,7 +4,7 @@ object EventList {
     val eventList = arrayListOf<Event>()
 
     fun getData(): ArrayList<Event> {
-        eventList.add(Event(isDaysSet = true, isDateSet = false, date = "", days = "M,T,W,R", time = "", timeSpecification = "All Day", period = ""))
+        eventList.add(Event(isDaysSet = true, isDateSet = false, date = "", days = "M,T,W,TH,R", time = "", timeSpecification = "All Day", period = ""))
```

Comment:
Isn't "R" for "Thursday"?  It looks like it is missing Friday, Saturday, and Sunday.

Reply:
I changed that for them. I added friday, saturday, and sunday.